src/App.scss

```diff
@@ -31,18 +32,18 @@
     margin: 0 5px;
     font-size: 40px;
     cursor: pointer;
-    border-top: 3px solid white;
+    border-top: 3px solid $white;
 
     &:hover {
-        color: #fff;
-        background: #04b87f;
-        border-top: 3px solid #067B93;
+        color: $white;
```

Comment:
Why do you use such names?
please use instead of `color:$white` `color:$font_color`